data/en/listtrim.json

```diff
@@ -4,14 +4,14 @@
 	"syntax": "listTrim(list [, delimiters])",
 	"returns": "string",
 	"related": [],
-	"description": "",
+	"description": "trims leading and trailing spaces from each list element",
 	"params": [
-		{"name": "list", "description": "", "required": true, "default": "", "type": "string", "values": []},
-		{"name": "delimiters", "description": "", "required": false, "default": ",", "type": "string", "values": []}
+		{"name": "list", "description": "a string list", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "delimiters", "description": "characters that separate list elements", "required": false, "default": ",", "type": "string", "values": []}
 	],
 	"engines": {
-		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/listtrim.html"}
+		"lucee": {"minimum_version": "", "deprecated": 4, "notes": "use ListCompact instead", "docs": "http://docs.lucee.org/reference/functions/listtrim.html"}
```

Comment:
Version properties such as `deprecated` should have string values.

Reply:
I copied that from pull request #945  The value is not a string in that change?

Reply:
Mea culpa!

Reply:
Ha Ha.  No problem. I updated the pull request.

Reply:
You should not consider unmerged PRs to be indicative of what is correct. There are many existing data files with this property that are all strings. You could also consider that it would just make sense for it to be the same as for any other version property (see `minimum_version`). I would ultimately advise you just look at the [schema](https://github.com/foundeo/cfdocs/blob/0e28a0793e734279cba49af8452b544d4b8beb0f/data/cfdocs.schema.json#L14) if you have any questions or doubts.

Reply:
Understood.

Reply:
Okay, you're right.